.github/workflows/enso4igv.yml

```diff
@@ -8,29 +8,127 @@ on:
     paths:
       - ".github/workflows/enso4igv.yml"
       - "tools/enso4igv/**/*"
+      - "engine/**/*"
+      - "lib/java/**/*"
+      - "lib/scala/**/*"
+      - "build.sbt"
 
 jobs:
-  build:
+  build_linux_parser:
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        java: ["11"]
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Java
-        uses: actions/setup-java@v2
+      - uses: actions/checkout@v4
+
+      - name: Install rustup
+        run: |
+          rustup target add x86_64-unknown-linux-musl
+
+      - name: Build Enso Parser
+        working-directory: .
+        env:
+          RUSTFLAGS: "-C target-feature=-crt-static"
+        run: |
+          cargo build -p enso-parser-jni -Z unstable-options --target x86_64-unknown-linux-musl --out-dir target/lib/
```

Comment:
One thing that might benefit from your review, @kazcw are these `cargo build` commands. Right now [they produce](https://github.com/enso-org/enso/actions/runs/9509862775?pr=7054) relatively huge artifacts:
![parser artifacts](https://github.com/enso-org/enso/assets/26887752/29fb33d8-2356-4cfa-8cef-15f88022f957)
and that's the size after compression, the [real size of the generated files](https://github.com/enso-org/enso/actions/runs/9509862775/job/26213518760#step:4:5) is:
```
 4229656  modules/lib/aarch64/libenso_parser.dylib
 2207232  modules/lib/enso_parser.dll
 4035816  modules/lib/x86_64/libenso_parser.dylib
 8151600  modules/lib/libenso_parser.so
```
Any chance to make to make these files smaller?

Reply:
`cargo build --release`

Reply:
Thank you. Trying in #10295